is-compatible/action.yml

```diff
@@ -36,20 +36,19 @@ runs:
         CODE=$(if [[ -n $(cat .levitate_output | grep "not fully compatible") ]]; then echo 1; else echo 0; fi)
         # Capture levitate output in an ENV variable
         IS_COMPATIBLE=$(cat .levitate_output)
-        # Github action specific: Escape special characters to save it in a github output context
-        IS_COMPATIBLE="${IS_COMPATIBLE//'%'/'%25'}"
-        IS_COMPATIBLE="${IS_COMPATIBLE//$'\n'/'%0A'}"
-        IS_COMPATIBLE="${IS_COMPATIBLE//$'\r'/'%0D'}"
         # Save outputs in github contexts
-        echo "::set-output name=is-compatible-stdout::$IS_COMPATIBLE"
-        echo "::set-output name=is-compatible-exit-code::$CODE"
+        echo "is-compatible-stdout<<EOF" >> $GITHUB_OUTPUT
```

Comment:
shouldn't this be:

`echo "is-compatible-stdout=$IS_COMPATIBLE" >> $GITHUB_OUTPUT`

Reply:
This doesn't work because `$IS_COMPATIBLE` is multilined string. This is the way you do it for multiline strings https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
got it